language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -23,11 +24,11 @@ const ALICE_TOKEN = computeToken(ALICE_PARTY);
 const BOB_PARTY = 'Bob';
 const BOB_TOKEN = computeToken(BOB_PARTY);
 
-const SANDBOX_PORT = 6865;
+let sandboxPort: number | undefined = undefined;
 const SANDBOX_PORT_FILE = 'sandbox.port';
-const JSON_API_PORT = 7575;
+let jsonApiPort: number | undefined = undefined;
 const JSON_API_PORT_FILE = 'json-api.port';
-const HTTP_BASE_URL = `http://localhost:${JSON_API_PORT}/`;
+const httpBaseUrl: () => string = () => `http://localhost:${jsonApiPort}/`
```

Comment:
This would have made it a bit terser:
```suggestion
const httpBaseUrl = (): string => `http://localhost:${jsonApiPort}/`;
```
And please add semicolons to the end of lines. It's not strictly necessary but I don't understand the implications of not doing it and am a bit wary hence.